ra/ra_test.go

```diff
@@ -550,9 +550,6 @@ func TestReuseAuthorizationDisabled(t *testing.T) {
 	_, sa, ra, _, cleanUp := initAuthorities(t)
 	defer cleanUp()
 
-	// Turn *off* AuthZ Reuse
```

Comment:
I think this assumed the `ra` had `reuseValidAuthz` enabled by default in the tests, which it isn't so this didn't actually do anything.